icubam/backoffice/handlers/operational_dashboard.py

```diff
@@ -0,0 +1,153 @@
+from absl import logging
+
+import numpy as np
+import tornado.web
+import math
+from collections import defaultdict
+
+from bokeh.embed import components
+from bokeh.plotting import figure
+from bokeh.transform import dodge
+from bokeh.models import ColumnDataSource
+
+from icubam.db.store import to_pandas
+from icubam.backoffice.handlers import base
+
+
+class OperationalDashHandler(base.BaseHandler):
+  ROUTE = '/operational-dashboard'
+
+  @tornado.web.authenticated
+  async def get(self):
```

Comment:
it is a very long function so i would split the logic of building the plot knowing that you have the correct dataFrame in another function and call it here. it would be more readable.